### DISCUSSION TITLE
Does cypress internally uses different node?

### DISCUSSION THREAD

**Discussion by karthik22061993** [cypress-io/cypress#14414] [Questions and Help]
Hello,

Gloablly I am using node version 14.0.0. I want to know If cypress internally uses different node version. If yes, Can someone please tell me how to check which version it is using. For end- to-end I am using Twilio and it is version dependent. Please let me know If more details are needed. Thanks

**Comment by bahmutov** [comment]
See https://docs.cypress.io/guides/references/configuration.html#Node-version

To find the Node version bundled with Cypress run

```
$ npx cypress version
Cypress package version: 6.2.1
Cypress binary version: 6.2.1
Electron version: 11.1.1
Bundled Node version: 12.18.3
```
